codecov.yaml

```diff
@@ -12,3 +12,5 @@ ignore:
   - "cli/mocks"
   - "assets"
   - "**/_test.go"
```

Comment:
```suggestion
```
Both of these should be fine to remove; assets is not included in the coverage (since it is outside cli folder) and `_test` files are automatically excluded by go test